CMakeLists.txt

```diff
@@ -5,24 +5,24 @@
 # lowest version supporting C_STANDARD property
 cmake_minimum_required (VERSION 3.1.3)
 
+# Set software information
+project(libsndfile VERSION 1.0.28.0)
+
```

Comment:
If you never use the `X.X.X.X` version, I can replace it with `X.X.X`.

The last zero from `PROJECT_VERSION` (aka `PROJECT_VERSION_TWEAK`) is used in the [version-metadata.rc.in.cmake](/evpobr/libsndfile/blob/fix-package-version/src/version-metadata.rc.in.cmake#L5). But it can be simply replaced by zero in place.